docs: update tips and tricks topic (#5571)
forcedotcom/salesforcedx-vscode: docs/_articles/en/vscode-desktop/tips-and-tricks.md

6.  The next time you’re working on a Salesforce DX project, you can launch VS Code using `code-sfdx` and all your extensions will be there.

You can have as many aliases as you want, with as many `extensions-dir` directories as you need to help organize your extensions.

## Create Keyboard Shortcuts for Common Tasks
If using keyboard shortcuts is your jam, you'll appreciate the rich keyboard shortcut editing experience with the Keyboard Shortcuts editor that VS Code provides. Open the editor from **Code > Settings > Keyboard Shortcuts** or from **Preferences: Open Keyboard Shortcuts** command (⌘K ⌘S). See [Key Bindings for Visual Studio Code](https://code.visualstudio.com/docs/getstarted/keybindings) for details.

## Use Quick Open  

> Quickly search and open files. using the keyboard shortcut ⌘P.

Reviewer: ```suggestion
Quickly search and open files using the keyboard shortcut ⌘P.
```

## Commands for Salesforce Development

> Open the Command Palette using the keyboard shortcut ⌘⇧P. Type "SFDX" to view a list of commands available in your Salesforce project.

Reviewer: Should this be ⌘P instead?

> Open the Command Palette using the keyboard shortcut ⇧⌘P. Type "SFDX" to view a list of commands available commands in your Salesforce project.

Reviewer 2: ```suggestion
Open the Command Palette using the keyboard shortcut ⌘⇧P. Type "SFDX" to view a list of commands available in your Salesforce project.
```

## Use the Extensions View 

> Bring up the Extensions view by clicking the Extensions icon in the Activity Bar on the side of VS Code or run the **View: Extensions** command (⇧⌘X). Search for an extension. On an extension's details page can look up a lot of information such as it's dependencies, feature details, settings, and change log.  

Reviewer 2: ```suggestion
Bring up the Extensions view by clicking the Extensions icon in the Activity Bar on the side of VS Code or run the **View: Extensions** command (⌘⇧X). Search for an extension. On an extension's details page you can look up a lot of information such as its dependencies, feature details, settings, and change log.  
```
